### DISCUSSION TITLE
DevOps Help

### DISCUSSION THREAD

**Discussion by irish1986** [Pattern-Projector/pattern-projector#225] [Ideas]
@courtneypattison 

My gf is into sewing and I've recently bought her a projector (epson brightlink 595wi). It has been a nightmare to setup until I've found your project.  Are you taking any contribution "DevOps" wise regarding automation for either code pipeline, unit test, dependancies management, etc..

I don't know much about sewing but I am very grateful for how easy this tool made setting up my GF projector and would like to give back.  I might whipped a few contribution and open PR if these are welcome.

**Comment by courtneypattison** [comment]
@irish1986 thank you! I'm glad you got her projector setup. I hope she's had a chance to cut something!

I'd appreciate the DevOps help since I have no experience with it other than a little CI on personal projects. I'm particularly interested in adding some visual testing (#30) into the flow since most of the functionality of the site is visual. I was thinking something like https://docs.cypress.io/guides/tooling/visual-testing What do you think?
